vscode/Makefile

```diff
@@ -0,0 +1,6 @@
+DEST := "~/.vscode/extensions/kite.vscode"
```

Comment:
actually no, but this is just for the dev workflow, and I do believe that at least on linux the path is the same (and windows doesn't even have `make` by default`)